FWCore/Framework/interface/Event.h

```diff
@@ -207,6 +207,9 @@ namespace edm {
     void
     getAllProvenance(std::vector<Provenance const*>& provenances) const;
 
+    void
```

Comment:
Please add a comment explaining what one can get with StableProvenance. (Or maybe what one can not get).